source/adminguide/templates.rst

```diff
@@ -413,6 +413,11 @@ When templates are deleted, the VMs instantiated from them will continue
 to run. However, new VMs cannot be created based on the deleted
 template.
 
+Cloudstack refuses to delete a template when VMs based on the template
+exist. If this is the case, the parameter "forced" has to be
+set to "true" to delete the template anyways. These VMs can no longer be
+reinstalled from that template, but will be unaffected otherwise.
+
```

Comment:
looks good, but I think the text should be integrated with the previous paragraph(s).

Reply:
start of the first paragraph
```
Templates may be deleted. However when the templates are used the default behaviour is to refuse deletion. In general, when a template spans multiple
Zones, only the copy that is selected ....
```
and hence
```suggestion
As said Cloudstack refuses to delete a template when VMs based on the template
exist. If this is the case, the parameter "forced" has to be
set to "true" to delete the template anyways. These VMs can no longer be
reinstalled from that template, but will be unaffected otherwise.

```

What do you think @MartinEmrich ?

Reply:
Yes, I tried to rephrase it a little. As long as it is factually correct, I like it that way.